Gemfile.lock

```diff
@@ -87,6 +87,3 @@ DEPENDENCIES
   mocha
   rack-test
   rake
-
-BUNDLED WITH
-   1.10.6
```

Comment:
Relatively recent policy on this:

- Everyone should be using latest bundler on all projects
- That is currently 1.10.6, so doing so will prevent this diff from cropping up
- If/when bundler does update, we'll see some noise here once per project

So... please `git reset` this against master, then `gem install bundler`, then you shouldn't see it vanish again on your next `bundle` invocation.